File: Tempo_de_Jogo.py
ini, fim = input().split()

ini = int(ini)
fim = int(fim)

if fim>ini:
    print("O JOGO DUROU {} HORA(S)".format(fim-ini))
elif fim<ini:
    fim+=24
    print("O JOGO DUROU {} HORA(S)".format(fim-ini))
else:
    print("O JOGO DUROU 24 HORA(S)")

File: Tempo_de_Jogo.js
const fs = require("fs");
const r = fs.readFileSync(0, "utf8");
let line = r.split(" ");
let i = parseInt(line[0]);
let f = parseInt(line[1]);
if(i >= f){
    console.log("O JOGO DUROU " + (24 - i + f) + " HORA(S)");
}else{
    console.log("O JOGO DUROU " + (f - i) + " HORA(S)");
}
